src/main/java/nextstep/courses/domain/Enrollment.java

```diff
@@ -0,0 +1,54 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.domain.sessionPolicy.SessionPolicyStrategy;
+import nextstep.payments.domain.Payment;
+import nextstep.users.domain.User;
+
+public class Enrollment {
+    private static final String NOT_FOUND_SESSION_ERROR_MESSAGE =
+            "수강 신청이 허용되지 않습니다. 결제 금액이 세션 가격과 일치하지 않습니다.";
+    public static final String MAX_CAPACITY_ERROR_MESSAGE = "강의 최대 수강 인원을 초과하였습니다.";
+    public static final String ENROLLMENT_ERROR_MESSAGE = "수강 신청 상태가 아닙니다.";
+    private final SessionStatus sessionStatus;
```

Comment:
```suggestion
    private static final String NOT_FOUND_SESSION_ERROR_MESSAGE =
            "수강 신청이 허용되지 않습니다. 결제 금액이 세션 가격과 일치하지 않습니다.";
    public static final String MAX_CAPACITY_ERROR_MESSAGE = "강의 최대 수강 인원을 초과하였습니다.";
    public static final String ENROLLMENT_ERROR_MESSAGE = "수강 신청 상태가 아닙니다.";

    private final SessionStatus sessionStatus;
```

상수와 멤버변수는 개행으로 구분지어주세요